<!-- -*- mode: XML -*- -->
<launch>
  <node pkg="uta_racecar" type="transmitter.py" name="teleop_twist_keyboard">
    <param name="ZED_IMAGE_WIDTH" type="int" value="1280"/>
    <param name="ZED_IMAGE_HEIGHT" type="int" value="720"/>
  </node>
</launch>
